### ISSUE TITLE
About learning rate

### ISSUE THREAD

**Issue by ShuangjunLiu** [ilovepose/DarkPose#20]
Hello. 
In paper, the learning rate is described as: 
"the base learning rate was fine-tuned to 2.5e-4, and decayed to 2.5e-5 and 2.5e-6 at the 90-th and 120-th epoch".  
In the repo, the initial learning rate is 0.001. 

Which one is better? Should I change it as what is described as in paper for reproducing?
